### PR TITLE
Moves to GitHub Actions publication

### PR DESCRIPTION
**Pull request by adriancole**
This moves off Travis and onto GitHub actions for all CI features.

This is the first Maven project, to do this in our org, so there might
be a glitch post-merge.

cc @openzipkin-contrib/core this is first try for a Maven project. The first Docker project was https://github.com/openzipkin/docker-java

### CODE REVIEW

**Comment by adriancole** [comment]
cc @reta

File: .github/workflows/create_release.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    tags: 'release-[0-9]+.[0-9]+.[0-9]+**'  # Ex. release-1.2.3
+
+jobs:
+  create_release:
+    runs-on: ubuntu-20.04  # newest available distribution, aka focal
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1  # only needed to get the sha label
```
Comment (adriancole) [inline .github/workflows/create_release.yml:19]:
```suggestion
          fetch-depth: 1  # only need the base commit as license check isn't run
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+[![Gitter chat](http://img.shields.io/badge/gitter-join%20chat%20%E2%86%92-brightgreen.svg)](https://gitter.im/openzipkin/zipkin)
```
Comment (adriancole) [inline README.md:1]:
```suggestion
[![Gitter chat](http://img.shields.io/badge/gitter-join%20chat%20%E2%86%92-brightgreen.svg)](https://gitter.im/openzipkin/secondary-sampling)
```

File: .github/workflows/create_release.yml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# yamllint --format github .github/workflows/create_release.yml
+---
+name: create_release
+
+# We create a release version on a trigger tag, regardless of if the commit is documentation-only.
+#
+# See https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet
+on:
+  push:
+    tags: 'release-[0-9]+.[0-9]+.[0-9]+**'  # Ex. release-1.2.3
```
Comment (anuraaga) [inline .github/workflows/create_release.yml:10]: Didn't realize this isn't a regex in the last PR 😢

**Review by anuraaga — APPROVED** [review]

File: .github/workflows/create_release.yml (hunk 3 above)
Comment (adriancole) [reply .github/workflows/create_release.yml:10]: show me a perfect proofreader and I'll show you a bridge for sale.. It's in Brooklyn!

**Comment by adriancole** [comment]
ok run looks clean now. let's see if it publishes a snapshot. thx @anuraaga

**Pull request merged**